pkgs/os-specific/linux/edac-utils/default.nix

```diff
@@ -1,32 +1,47 @@
-{ lib, stdenv, fetchFromGitHub, perl, makeWrapper
+{ lib, stdenv, fetchFromGitHub, perl
 , sysfsutils, dmidecode, kmod }:
 
 stdenv.mkDerivation {
   pname = "edac-utils";
-  version = "unstable-2015-01-07";
+  version = "unstable-2023-01-30";
 
   src = fetchFromGitHub {
     owner = "grondo";
     repo = "edac-utils";
-    rev = "f9aa96205f610de39a79ff43c7478b7ef02e3138";
-    sha256 = "1dmfqb15ffldl5zirbmwiqzpxbcc2ny9rpfvxcfvpmh5b69knvdg";
+    rev = "8fdc1d40e30f65737fef6c3ddcd1d2cd769f6277";
+    hash = "sha256-jZGRrZ1sa4x0/TBJ5GsNVuWakmPNOU+oiOoXdhARunk=";
   };
 
-  nativeBuildInputs = [ perl makeWrapper ];
-  buildInputs = [ sysfsutils ];
+  # Hard-code program paths instead of using PATH lookups. Also, labels.d and
+  # mainboard are for user-configurable data, so do not look for them in Nix
+  # store.
+  patches = [ ./edac-ctl.patch ];
+  postPatch = ''
+    substituteInPlace src/util/edac-ctl.in \
+      --subst-var-by dmidecode ${lib.getExe' dmidecode "dmidecode"} \
+      --subst-var-by modprobe ${lib.getExe' kmod "modprobe"} \
+      --replace '"$sysconfdir/edac/labels.d"' '"/etc/edac/labels.d"' \
+      --replace '"$sysconfdir/edac/mainboard"' '"/etc/edac/mainboard"'
```

Comment:
Would it make sense to use `--replace-fail` here? Then we would notice when the upstream changes to that this no longer works.